src/workers/webdav/VirtualDrive.ts

```diff
@@ -1,5 +1,5 @@
 import Logger from 'electron-log';
-import { exec } from 'child_process';
+import { exec, execSync } from 'child_process';
```

Comment:
i think this new import is not being used

Reply:
yep, solved